### PR TITLE
BinaryOpBetweenNumberAndStringRector don't trust phpdoc

### PR DESCRIPTION
**Pull request by staabm**

### CODE REVIEW

File: rules/Php71/Rector/BinaryOp/BinaryOpBetweenNumberAndStringRector.php [hunk 1]
```diff
@@ -100,15 +100,15 @@ public function refactor(Node $node): ?Node
             return null;
         }
 
-        if ($this->isStringOrStaticNonNumbericString($node->left) && $this->nodeTypeResolver->isNumberType(
+        if ($this->isStringOrStaticNonNumericString($node->left) && $this->nodeTypeResolver->isNumberType(
```
Comment (staabm) [inline rules/Php71/Rector/BinaryOp/BinaryOpBetweenNumberAndStringRector.php:103]: just fixing a typo in the method name

**Review by samsonasik — APPROVED** [review]

**Comment by samsonasik** [comment]
Thank you @staabm

**Comment by staabm** [comment]
@TomasVotruba any plans for a new release?

**Comment by TomasVotruba** [comment]
Not direct yet as not much news. Expect in about week or two :+1:

**Pull request merged**